src/components/SidebarNavigationItem/SidebarNavigationItem.tsx

```diff
@@ -87,26 +139,56 @@ const Wrapper = styled.div`
       color: inherit;
     }
   }
+  &:hover [data-trigger-icon],
+  [data-open="true"][data-trigger-icon] {
+    visibility: visible;
+  }
 `;
-const IconsWrapper = styled.div`
-  display: flex;
-`;
+
 const CollapsibleNavigationItem = ({
-  collapsible,
-  icon,
   label,
   children,
-}: CollapsibleSidebarNavigationItemProps) => (
-  <CollapsibleWrapper>
-    {label && collapsible && (
-      <SidebarAccordion title={label} icon={icon} iconSize="small">
-        {children}
-      </SidebarAccordion>
-    )}
-  </CollapsibleWrapper>
-);
+  open,
+  onOpenChange,
+  iconDir = "left",
+  icon,
+  level,
+  selected,
+  ...props
+}: CollapsibleSidebarNavigationItemProps & { level: number }) => {
+  if (!label) {
+    return;
+  }
+  return (
+    <Collapsible
+      open={open}
+      onOpenChange={onOpenChange}
+    >
+      <Wrapper
+        as={Collapsible.Header}
```

Comment:
same as above 

maybe rename Wrapper to something more meaningful like `Header`?

```js
const Header = styled(Collapsible.Header)``
```